### PR TITLE
Enable Python alternatives when running tests with --skip-custom-ops

### PR DESCRIPTION
**Pull request by guillaumekln**

### CODE REVIEW

**Comment by guillaumekln** [comment]
Any idea why `from tensorflow_addons import options` is failing in Bazel tests? I also got that in my other PR.

File: tensorflow_addons/utils/test_utils.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 import pytest
 import tensorflow as tf
 
+from tensorflow_addons import options
```
Comment (bhack) [inline tensorflow_addons/utils/test_utils.py:24]: if you use https://github.com/tensorflow/addons/blob/master/tensorflow_addons/options.py#L36-L40 seems it works.

**Comment by bhack** [comment]
To import `options` in bazel you need to add `"//tensorflow_addons:options.py",` in https://github.com/tensorflow/addons/blob/master/tensorflow_addons/utils/BUILD#L8

**Comment by guillaumekln** [comment]
Ah thanks @bhack, I forgot about that.

I'm going to close this PR though, we might want to do it differently.

**Pull request closed without merging**